README.md

```diff
@@ -1,5 +1,7 @@
 # Introduction
 
+*** We are modernizing the quantum katas experience.   Please visit https://quantum.microsoft.com/experience/quantum-katas to try the new online Azure Quantum katas experience, with integrated assistance from Copilot in Azure Quantum. ***  
```

Comment:
Three asterisks don't highlight the text. We want this to stand out, I'd recommend `>` in the beginning of the line (the note will look like the "Note that mybinder.org..." paragraph) or double asterisks (without spaces between them and text) to make it bold, or both

Reply:
updated per your recommendations!